parm/soca/obs/obs_list.yaml

```diff
@@ -1,6 +1,6 @@
 observers:
 - !INC ${OBS_YAML_DIR}/adt_j3.yaml
-- !INC ${OBS_YAML_DIR}/salt_profile_fnmoc.yaml
+#- !INC ${OBS_YAML_DIR}/salt_profile_fnmoc.yaml
```

Comment:
This change was committed at 53b99c6 with the comment `removing broken ufo `.  Just curious, are there plans to fix the ufo and reactivate this line?

Reply:
yes, and I'm not sure the failure is related to the recent changes or even this repo. I added an issue a few days back.